smash.js

```diff
@@ -16,8 +16,56 @@ function replaceAllImages(){
+    'https://media2.fdncms.com/sfweekly/imager/chef-guy-fieri-and-smash-mouth-lead-singer/u/big/2611979/smashfieri_opt.jpg',
+    'http://img.cache.vevo.com/Content/VevoImages/video/E940A9FC7080BE3A954DB49FDDCF3635.jpg',
+    'http://www.billboard.com/files/media/smash-mouth-1999-billboard-650.jpg',
+    'http://pedestriantv-prod.s3.amazonaws.com/images/article/2016/06/15/shreksmashmouth2-619-386.jpg',
+    'https://i.ytimg.com/vi/J48dqyz_C6s/maxresdefault.jpg',
+    'http://static.stereogum.com/uploads/2016/10/423presspassweb-1477781306.jpg',
+    'http://www.thrashhits.com/wpress/wp-content/uploads/2011/10/SmashmouthEggsFAILUREThrashHits.jpg',
+    'http://r.fod4.com/c=sq/s=w900,pd1/o=80/http://p.fod4.com/p/media/6ac8c81954/W5ZJxzygQCSc4hZG3p86_smash%20square.jpg',
+    'http://vegasexperience.com/wp-content/uploads/2016/04/smash_mouth_horizontal.jpg',
+    'https://m.elitestatic.com/m/658e03cb1d43b669/800x400-elite-daily-Guy-Fieri.jpg',
+    'http://cdn2.crushable.com/wp-content/uploads/2011/08/tumblr_lqpakfnQ2E1qzy8n8o1_500.jpg',
+    'http://cdn.newsapi.com.au/image/v1/da3e9bf4bdc76e634f300a300137f0ae',
+    'http://www.oocities.org/sunsetstrip/lounge/1638/hrbs_2.jpg',
+    'https://ewedit.files.wordpress.com/2016/10/screen-shot-2016-10-21-at-12-29-08-pm.jpg?w=612',
+    'http://www.grandslamcharityjam.org/wp-content/uploads/2011/02/Smash-Mouth-PHOTO-1.jpg',
+    'https://img.grouponcdn.com/deal/f0a89c78a56140149cf441fb9dbee787/87/v1/c700x420.jpg',
+    'https://images.vice.com/noisey/content-images/contentimage/no-slug/90e745bf4bd5ec85062d121f19cbac8d.jpg',
+    'https://cbs931jackfm.files.wordpress.com/2014/04/smash-mouth-brenda-chase-getty-images.jpg?w=640&h=360&crop=1',
+    'https://s-media-cache-ak0.pinimg.com/originals/e0/0f/de/e00fde21dee2a075d2c4456b97a8af4a.jpg',
+    'https://timedotcom.files.wordpress.com/2016/07/smash-mouth-group-pic.jpg?quality=85',
+    'http://static.tvtropes.org/pmwiki/pub/images/smashmouth_8671.jpg',
+    'https://metrouk2.files.wordpress.com/2016/06/111154933.jpg?w=748&h=494&crop=1',
+    'http://www.altpress.com/images/uploads/news/underoath_SM.jpg',
+    'http://diffuser.fm/files/2016/06/Smash-Mouth.jpg?w=630&h=420&zc=1&s=0&a=t&q=89',
+    'http://cdn.boulevards.com/imgs/event_imgs/20150609184150-SmashMouth2EFM.jpg',
+    'http://www.aceshowbiz.com/images/photo/smash_mouth.jpg',
+    'http://i.dailymail.co.uk/i/pix/2016/08/29/22/37B1119900000578-3764207-image-a-56_1472507455538.jpg', 'http://www.inforum.com/sites/default/files/styles/16x9_620/public/0B5WzNj8WWJ6_djhvdlJKY1AyYkk.jpg?itok=W3oRWYnu',
+    'https://thumbs.mic.com/MzgwODkwZTY0NSMvX0FJaU1WQ2J0NEM0Zkp1clJZZUg5Y2xxRXBzPS8weDE2NToyOTE2eDE1NzcvMTYwMHg5MDAvZmlsdGVyczpmb3JtYXQoanBlZyk6cXVhbGl0eSg4MCkvaHR0cHM6Ly9zMy5hbWF6b25hd3MuY29tL3BvbGljeW1pYy1pbWFnZXMvdWFrZXBmZ2lldWNlZG1tczJucXd3ZmkzZzhteWEzYmdocXR2eW9kdmRhY3p3Z3dzNnVocGJveGQ2ZDdoNXBteS5qcGc.jpg',
+    'http://www.pngmart.com/image/25428',
```

Comment:
The url http://www.pngmart.com/image/25428 doesn't point to the image file. Maybe it should be http://www.pngmart.com/files/3/Shrek-PNG-Photos.png ?